delta/rdf/src/main/scala/ch/epfl/bluebrain/nexus/delta/rdf/shacl/ValidationReport.scala

```diff
@@ -38,8 +38,9 @@ object ValidationReport {
 
   private val shaclCtx: ContextValue = ContextValue(contexts.shacl)
 
-  final def apply(report: Resource)(implicit api: JsonLdApi, rcr: RemoteContextResolution): IO[ValidationReport] = {
-    val tmpGraph = Graph.unsafe(DatasetFactory.create(report.getModel).asDatasetGraph())
+  final def apply(report: Resource)(implicit rcr: RemoteContextResolution): IO[ValidationReport] = {
+    implicit val api: JsonLdApi = JsonLdJavaApi.lenient
```

Comment:
We only need to use the configured `JsonLdApi` for write operations of resources.
For read operations or like in this case, it is generated by an underlying library, there is no reason to be strict and enforce extra checks